### PR TITLE
Source Sentry: turn on default availability strategy

### PR DESCRIPTION
**Pull request by davydov-d**
## What
https://github.com/airbytehq/oncall/issues/1383

## How
Use default availability strategy not to fail syncs when 403 error is risen

### CODE REVIEW

**Review by lazebnyi — APPROVED** [review]

File: airbyte-integrations/connectors/source-sentry/source_sentry/streams.py [hunk 1]
```diff
@@ -26,10 +25,6 @@ def __init__(self, hostname: str, **kwargs):
     def url_base(self) -> str:
         return self._url_base
 
-    @property
-    def availability_strategy(self) -> Optional["AvailabilityStrategy"]:
```
Comment (roman-yermilov-gl) [inline airbyte-integrations/connectors/source-sentry/source_sentry/streams.py:30]: As I understand from oncall comments it needs `availability_strategy` to be turned on by default for this connector. But if to just remove the property from the stream, then nothing to be called by default. It must be specified explicitly for something to happen

**Review by roman-yermilov-gl — CHANGES_REQUESTED** [review]

File: airbyte-integrations/connectors/source-sentry/source_sentry/streams.py (hunk 1 above)
Comment (davydov-d) [reply airbyte-integrations/connectors/source-sentry/source_sentry/streams.py:30]: The base class defines this property, so I just removed the override

Comment (roman-yermilov-gl) [reply airbyte-integrations/connectors/source-sentry/source_sentry/streams.py:30]: Right, they add it to the base class yesterday

**Review by roman-yermilov-gl — APPROVED** [review]

**Review by pedroslopez — APPROVED** [review]

**Pull request merged**